main.go

```diff
@@ -322,8 +318,25 @@ func main() {
 					projName := c.Args().Get(0)
 					err := vscodesupport.OpenCode(
 						projName, usrHome+confDir+wsFName)
-					lib.Check(err)
-					return nil
+					return err
+				},
+			},
+			{
+				Name:    "finder",
+				Aliases: []string{"open"},
+				Description: `
+				Opens selected project in a finder window.
+				This is targeted towards macOS users.
+				Argument should be a project name.`,
+				Usage: "Opens selected project in a finder window",
+				Action: func(c *cli.Context) error {
+					if qFlag {
+						fmt.Println("(okay, I'll try to be quiet.)")
+					}
+					projName := c.Args().Get(0)
+					err := finder.OpenFinder(
+						projName, usrHome+confDir+wsFName)
+					return err
```

Comment:
Might need to check here that `c.Args().Len() >= 1` (or `==1`) to prevent null pointer dereference if the user does `canaveral finder` with no argument.

Reply:
This is covered in a test case (null pointers actually get handled as empty strings, don't ask me why). I also just manually tested it to see if the null pointer is an issue, and it looks good. Thanks for highlighting this.